<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btn_convert">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Convert</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_none">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>none</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_semicol">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>;</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtn_sinf">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SINF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_sep_head">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Separator:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_selectfile">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>select file</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file_out">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>file</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_status_out">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>240</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file_output">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>output file</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_file_head">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>551</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Acceped file types: .bmp, .png, .jpg</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>724</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
